CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * Fixed a crash during turn-by-turn navigation. ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820))
 * Fixed a crash that could happen while simulating a route. ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820))
 * Fixed an issue causing MapboxVoiceController to speak instructions using VoiceOver instead of the Mapbox Voice API. ([#1830](https://github.com/mapbox/mapbox-navigation-ios/issues/1830))
+* Added `RouterDelegate.router(_:willArriveAt:in:distance:)` to assist with preparations for arrival. ([#1847](https://github.com/mapbox/mapbox-navigation-ios/pull/1847))
```

Comment:
```suggestion
* Added `RouterDelegate.router(_:willArriveAt:after:distance:)` to assist with preparations for arrival. ([#1847](https://github.com/mapbox/mapbox-navigation-ios/pull/1847))
```

Reply:
will change to `NavigationViewControllerDelegate.navigationViewController(_:willArriveAt:after:distance:)` since it's more relevant for most users.